README.md

```diff
@@ -138,17 +128,23 @@ Mastodon is **free, open-source software** licensed under **AGPLv3**.
 
 You can open issues for bugs you've found or features you think are missing. You can also submit pull requests to this repository or submit translations using Crowdin. To get started, take a look at [CONTRIBUTING.md](CONTRIBUTING.md). If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
 
-**IRC channel**: #mastodon on irc.libera.chat
+**IRC channel**: #mastodon on [`irc.libera.chat`](https://libera.chat)
 
 ## License
 
-Copyright (C) 2016-2024 Eugen Rochko & other Mastodon contributors (see [AUTHORS.md](AUTHORS.md))
+Copyright (c) 2016-2024 Eugen Rochko (+ [`mastodon authors`](AUTHORS.md))
+
+Licensed under GNU Affero General Public License as stated in the [LICENSE](LICENSE):
+
+```
```

Comment:
If we're going to convert this to a pre/code block - can you hard wrap the contents at 80 cols?

Reply:
It won't look as good but sure